.rubocop.yml

```diff
@@ -14,8 +15,8 @@ Style/StringLiterals:
 Layout/LineLength:
   Max: 120
 Layout/HashAlignment:
-  EnforcedHashRocketStyle: table
-  EnforcedColonStyle: table
+  EnforcedHashRocketStyle: key 
+  EnforcedColonStyle: key 
```

Comment:
I personally find table layout more readable than key. In any case, I'm not sure why you are suggesting a change to the style of this gem.